app/src/main/java/com/example/androidacademymsk/MoviesAdapter.kt

```diff
@@ -30,15 +36,26 @@ class MoviesAdapter : RecyclerView.Adapter<MoviesAdapter.MovieViewHolder>() {
     }
 
     class MovieViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
-        private val title: TextView? = itemView.findViewById(R.id.movie_title)
-        private val picture: ImageView? = itemView.findViewById(R.id.movie_picture)
+        private val title: TextView = itemView.findViewById(R.id.movieTitle)
+        private val picture: ImageView = itemView.findViewById(R.id.moviePicture)
+        private val ratingBar: MaterialRatingBar = itemView.findViewById(R.id.movieRatingBar)
+        private val pg: TextView = itemView.findViewById(R.id.pg)
 
         fun setData(movieCard: MovieCard) {
-            Glide.with(itemView.context)
-                .load(R.drawable.mock_picture_movie)
-                .into(picture)
+            val multiTransformation = MultiTransformation(
+                CenterCrop(),
+                RoundedCornersTransformation(14, 1, RoundedCornersTransformation.CornerType.TOP)
+            )
```

Comment:
I'm not using Glide and won't tell you what the problem is, but you don't want the rounded corner at the bottom of the image.
![image](https://user-images.githubusercontent.com/10943612/101981870-0eb32000-3c81-11eb-86b1-45c972364e6e.png)